product_docs/docs/edbcloud/beta/using_cluster/06_analyze_with_superset.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Analyzing Your Data with Apache Superset"
+---
+You can use Apache Superset to analyze, explore, and visualize data stored in your PostgreSQL clusters.
+You can open Superset from the [EDB Cloud](https://portal.edbcloud.com/) portal using the **Analyze** links. 
+
+Default permissions include the ability to view data and create dashboards. To add or alter data sources, you need additional permissions. For more information on permissions to access Superset, see [Managing Superset Access](../administering_cluster/01a_superset_access).
+
+!!! Note
+    Apache Superset shares Single Sign On with EDB Cloud. If you log out of Superset you will be logged out of EDB Cloud and vice-versa.
+
+
+## Connecting Superset to your Cluster
+
+
+
+
+You need your host and port to connect to Superset. To find the host and port of your cluster:
```

Comment:
Nit: they need more than host and port - they need their username, password, dbname, etc., right?

Reply:
Right, in addition we could even use a word `valid SQLAlchemy connection string`. Should be self explanatory for users methinks .

Reply:
Well, I don't think we should expect that anyone knows what SQLAlchemy is...

Reply:
See my latest commit.